sql/sys_vars.cc

```diff
@@ -4010,6 +4010,14 @@ static Sys_var_charptr Sys_secure_file_priv(
     CMD_LINE(REQUIRED_ARG), IN_FS_CHARSET,
     DEFAULT(DEFAULT_SECURE_FILE_PRIV_DIR));
 
+static Sys_var_charptr Sys_secure_log_path(
+    "secure_log_path",
+    "Limit location of general log, slow log and buffered error log"
+    "within specified directory",
+    READ_ONLY NON_PERSIST GLOBAL_VAR(opt_secure_log_path),
+    CMD_LINE(REQUIRED_ARG), IN_FS_CHARSET,
+    DEFAULT(DEFAULT_SECURE_FILE_PRIV_DIR));
+
```

Comment:
@dutow Change looks good. Please add some negative tests with buffered_error_log and the sys_vars.test